src/components/Form/Field.js

```diff
@@ -111,6 +114,12 @@ const arrowDownStyle = css({
   top: lineHeight + fieldHeight / 2 - 3,
   cursor: 'pointer'
 })
+const iconWrapperStyle = css({
+  position: 'absolute',
+  right: 3,
+  top: fieldHeight - (fieldHeight * .75 / 2),
```

Comment:
better definition: `lineHeight + 5` or `lineHeight + fieldHeight  * 0.25 / 2`

`lineHeight` is the label hight (top padding of the `<label>`, but absolute ignores padding).

Reply:
(currently this only works because `lineHeight * 2 === fieldHeight`)